packages/query/src/block-processor.ts

```diff
@@ -8,8 +8,9 @@ import { sha256Hash } from '@penumbra-zone/crypto-web';
 import {
   BlockProcessorInterface,
   IndexedDbInterface,
-  StoreCommitment,
   ViewServerInterface,
+  hexToUint8Array,
+  uint8ArrayToHex,
```

Comment:
previous versions of this PR simply matched on objects, but now we are performing more map lookups and using more equivalent objects from different sources. javascript cannot compare uint8 for equality so they must all be stringified